### PR TITLE
Mint: Remember values

### PR DESCRIPTION
**Pull request by tarwin**
When coming back from Preview, remember values in inputs.

### CODE REVIEW

File: src/pages/mint/index.js [hunk 1]
```diff
@@ -119,34 +119,42 @@ export const Mint = () => {
               <Input
                 type="text"
                 onChange={(e) => setTitle(e.target.value)}
-                placeholder="title"
```
Comment (andrevenancio) [inline src/pages/mint/index.js:122]: Is there a reason why you're changing the <Input /> API and removing the placeholder text? with API changes like this I would like to get a rational on why we need to change it?

File: src/pages/mint/index.js [hunk 2]
```diff
@@ -119,34 +119,42 @@ export const Mint = () => {
...
+                label="TITLE"
+                value={title}
               />
 
               <Input
                 type="text"
                 onChange={(e) => setDescription(e.target.value)}
-                placeholder="description"
+                label="DESCRIPTION"
```
Comment (andrevenancio) [inline src/pages/mint/index.js:129]: also, including new text will force me to chase everyone who contributed towards the translations. why do we need this?

File: src/pages/mint/index.js [hunk 3]
```diff
@@ -119,34 +119,42 @@ export const Mint = () => {
...
                 onChange={(e) => setAmount(e.target.value)}
-                placeholder="amount"
+                label="QUANTITY"
+                helpText="number of OBJKTs to mint"
+                value={amount}
               />
 
               <Input
-                type="royalties"
+                type="number"
```
Comment (andrevenancio) [inline src/pages/mint/index.js:151]: This I like! Was that a bug? oops.

Comment (tarwin) [reply src/pages/mint/index.js:151]: Yeah, I'm guessing it was a bug :D

File: src/pages/mint/index.js (hunk 1 above)
Comment (tarwin) [reply src/pages/mint/index.js:122]: Good question. "Placeholder" has a specific meaning in HTML inputs, which is what it was used for before. You can still use placeholder how it was used earlier. I just added `label` which adds a `<Label/>` around the input.

File: src/pages/mint/index.js (hunk 2 above)
Comment (tarwin) [reply src/pages/mint/index.js:129]: I'm happy to chase down the translations - I should just look at who commited translation changes right? I wasn't able to work out how the translations work actually so thought maybe they won't being used yet.

**Comment by andrevenancio** [comment]
This is a duplicate of the PR that got submitted and merged https://github.com/hicetnunc2000/hicetnunc/pull/416

**Pull request closed without merging**